src/py/migraphx_py.cpp

```diff
@@ -20,6 +22,94 @@
 using half   = half_float::half;
 namespace py = pybind11;
 
+namespace migraphx {
+migraphx::value kwargs_to_value(py::kwargs kwargs);
+
+migraphx::value list_to_value(py::list lst)
```

Comment:
Should be named `migraphx_to_value` so it will be called automatically by `migraphx::to_value`.